### PR TITLE
Set sequential tests stages in non-komodo jenkins pipeline as paralle…

### LINKED ISSUE #1812: Set sequential tests stages in non-komodo jenkins pipeline as parallel
**Problem description.**
Currently [ctests](https://github.com/equinor/ert/blob/47718c11af96c08e04fd295dc9b7db4187e5cf02/Jenkinsfile-libres#L24), [equinor tests](https://github.com/equinor/ert/blob/47718c11af96c08e04fd295dc9b7db4187e5cf02/Jenkinsfile-libres#L29) and [normal-pytest](https://github.com/equinor/ert/blob/47718c11af96c08e04fd295dc9b7db4187e5cf02/Jenkinsfile-libres#L34)
are run sequentially, but this can be done parallelly to speed up the total run of the pipeline

**Solution**
Implement a declarative Jenkins parallel structure for the before mention stages

**Additional information**
possible starting point
```
        stage('tests') {
            failFast true
            parallel {
                stage('run ctest') {
                        steps {
                            sh 'sh libres/testjenkins.sh run_ctest'
                        }
                    }
                stage('run pytest_equinor') {
                        steps {
                            sh 'sh libres/testjenkins.sh run_pytest_equinor'
                        }
                    }
                stage('run pytest_normal') {
                        steps {
                            sh 'sh libres/testjenkins.sh run_pytest_normal'
                        }
                }
            }
        }
```

### PR DESCRIPTION
**Pull request by abacus442**
Set sequential tests stages in non-komodo jenkins pipeline as parallel

**Issue**
Resolves #1812

**Approach**
Implemented a declarative Jenkins parallel structure for the tests

### CODE REVIEW

**Comment by ertomatic** [comment]
Can one of the admins verify this patch?

**Comment by jondequinor** [comment]
test this please

**Comment by jondequinor** [comment]
Assuming @abacus442 is really @sregales-TNO ? Are you intending of using this more, I can add you to the list of approveds.

**Comment by abacus442** [comment]
@jondequinor   Nope, newbie here

**Comment by jondequinor** [comment]
Sorry for the mix up 😅 I'll get back to you with logs of what failed.

**Comment by jondequinor** [comment]
test this please

**Comment by jondequinor** [comment]
```
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
WorkflowScript: 50: expecting '}', found '' @ line 50, column 1.
1 error

	at org.codehaus.groovy.control.ErrorCollector.failIfErrors(ErrorCollector.java:310)
	at org.codehaus.groovy.control.ErrorCollector.addFatalError(ErrorCollector.java:150)
	at org.codehaus.groovy.control.ErrorCollector.addError(ErrorCollector.java:120)
	at org.codehaus.groovy.control.ErrorCollector.addError(ErrorCollector.java:132)
	at org.codehaus.groovy.control.SourceUnit.addError(SourceUnit.java:350)
	at org.codehaus.groovy.antlr.AntlrParserPlugin.transformCSTIntoAST(AntlrParserPlugin.java:144)
	at org.codehaus.groovy.antlr.AntlrParserPlugin.parseCST(AntlrParserPlugin.java:110)
	at org.codehaus.groovy.control.SourceUnit.parse(SourceUnit.java:234)
	at org.codehaus.groovy.control.CompilationUnit$1.call(CompilationUnit.java:168)
	at org.codehaus.groovy.control.CompilationUnit.applyToSourceUnits(CompilationUnit.java:943)
	at org.codehaus.groovy.control.CompilationUnit.doPhaseOperation(CompilationUnit.java:605)
	at org.codehaus.groovy.control.CompilationUnit.processPhaseOperations(CompilationUnit.java:581)
	at org.codehaus.groovy.control.CompilationUnit.compile(CompilationUnit.java:558)
	at groovy.lang.GroovyClassLoader.doParseClass(GroovyClassLoader.java:298)
	at groovy.lang.GroovyClassLoader.parseClass(GroovyClassLoader.java:268)
	at groovy.lang.GroovyShell.parseClass(GroovyShell.java:688)
	at groovy.lang.GroovyShell.parse(GroovyShell.java:700)
	at org.jenkinsci.plugins.workflow.cps.CpsGroovyShell.doParse(CpsGroovyShell.java:142)
	at org.jenkinsci.plugins.workflow.cps.CpsGroovyShell.reparse(CpsGroovyShell.java:127)
	at org.jenkinsci.plugins.workflow.cps.CpsFlowExecution.parseScript(CpsFlowExecution.java:561)
	at org.jenkinsci.plugins.workflow.cps.CpsFlowExecution.start(CpsFlowExecution.java:522)
	at org.jenkinsci.plugins.workflow.job.WorkflowRun.run(WorkflowRun.java:337)
	at hudson.model.ResourceController.execute(ResourceController.java:97)
	at hudson.model.Executor.run(Executor.java:428)
Finished: FAILURE
```

Seems you're missing a final  `}` at line 50. The indentation of that file is a bit off.

File: Jenkinsfile-libres [hunk 1]
```diff
@@ -46,4 +46,4 @@ pipeline {
             deleteDir()
         }
     }
-}
+}
```
Comment (jondequinor) [inline Jenkinsfile-libres]: Could you add a trailing newline?

**Comment by jondequinor** [comment]
test this please

**Comment by sregales-TNO** [comment]
Is there any explanation for why this issue was closed?

**Comment by markusdregi** [comment]
Seems like the user was deleted as well... 🤔 That might very well be the reason it was closed.

The issue is back in the backlog and it was probably not a great issue to solve for an external developer anyway. I suggest we leave it as is and pick this issue up again ourselves when it hits the top of our list :)

**Pull request closed without merging**